app/controllers/katello/api/v2/srpms_controller.rb

```diff
@@ -0,0 +1,17 @@
+module Katello
+  class Api::V2::SrpmsController < Api::V2::ApiController
+    apipie_concern_subst(:a_resource => N_("a source repository"), :resource => "srpms")
+    include Katello::Concerns::Api::V2::RepositoryContentController
+
+    api :GET, "/srpms", N_("List srpms")
+    param :organization_id, :number, :desc => N_("Organization identifier")
+    param :content_view_version_id, :number, :desc => N_("Content View Version identifier")
+    param :content_view_filter_id, :number, :desc => N_("Content View Filter identifier")
+    param :repository_id, :number, :desc => N_("Repository identifier")
+    param :environment_id, :number, :desc => N_("Environment identifier")
+    param_group :search, Api::V2::ApiController
+    def index
+      super
+    end
+  end
+end
```

Comment:
i'd probably do something similar to the packages_controller here as far as sorting goes:
```
    def default_sort
      lambda { |query| query.default_sort }
    end
```

which means we'd probably want to copy the default sort method to srpm.rb too (from rpm.rb)